bucket/librewolf.json

```
@@ -26,7 +27,11 @@
             "LibreWolf"
         ]
     ],
-    "persist": "Profiles",
+    "persist": [
+        "Profiles",
+        "LibreWolf\\defaults\\prefs",
+        "LibreWolf\\*.js"
```

```suggestion
        "LibreWolf\\defaults\\prefs"
```

**persist** does not support wildcards.
If order to save `*.js`, we need to copy it in `pre_install` and `pre_uninstall` like:

```
"pre_install": [
    "......",
    "if (Test-Path \"$dir\\Librewolf\\*.js\") { Copy-Item \"$dir\\Librewolf\\*.js\" \"$persist_dir\\\" }"
]
"pre_uninstall": "if (Test-Path \"$persist_dir\\Librewolf\\*.js\") { Copy-Item \"$persist_dir\\Librewolf\\*.js\" \"$dir\\\" }"
```